scripts/run_all_benchmarks.sh

```diff
@@ -26,7 +26,7 @@ function run_benchmark() {
   --template=$3
 }
 
-cargo build --profile production --features runtime-benchmarks --features frequency --workspace || exit_err
+cargo build --profile production --features runtime-benchmarks --features all-frequency-features --workspace || exit_err
```

Comment:
why dont we add this line to the Makefile? that way we reference it one place for other build scripts

Reply:
I need to check with DevOps if they are calling this script as is else they will have to update Jenkins job for the same too

Reply:
@saraswatpuneet It uses `make benchmarks`

Reply:
@wilwade  yes, question should be build all features for benchmarks or just frequency like it was before this PR